### DISCUSSION TITLE
Support indexing multiple records from a single element

### DISCUSSION THREAD

**Discussion by zizther** [studioespresso/craft-scout#287] [General]
Is there a way to support indexing multiple records from a single element when saving entries in Craft.

My use-case is that I have a plain text field which contains a comma separated list which I want to loop through and add each to an index as individual records.

**Comment by janhenckens** [comment]
You could try to set up separate indexes (and index configurations), which match the samen element criteria, and have the transformer return the correct item for the index in question.

Although a comma separated string might be a bit too loosely coupled to base this on?

**Reply by zizther** [reply]
Thanks. I did try this, but wasn't able to loop through the comma separated list and add them individually as records to the index.

Have you any suggestions on how this could be applied via the scout config?
The issue I am facing is that the transform is really setup to expect one item I presume.

My other thought was to manage this from a custom module and connect with Scout to send the data to Algolia.

**Reply by janhenckens** [reply]
Scout uses the Algolia PHP client so you could use that - but you'll have to configure it again outside of Scout, as the plugin is really only set up to use through the indexes.
